README.md

```diff
@@ -58,5 +58,32 @@ export CUBIT_ROOT=path_to_cubit_root_directory
 To check if everything worked as expected, run the tests from within the `tests` directory
 ```bash
 cd path_to_cubitpy/tests
-pytest -q testing.py 
+pytest -q testing.py
 ```
+
+If you intend to actively develop CubitPy, please make sure to install the `pre-commit` hook within the python environment to follow our style guides:
+```bash
+pre-commit install
+```
```

Comment:
The same statement comes also just a couple of lines below under "Code Standards".

Reply:
I know, also the part on testing. I chose it this way, in case someone installs CubitPy and then only reads the contributing when starting the first contribution.

Reply:
I am also not fully satisfied with the current approach. But I prefer the current one over having the part on testing / pre_commit hooks just once.